packages/visx-demo/src/pages/linktypes.tsx

```diff
@@ -1,26 +1,26 @@
-import React from 'react';
-import Show from '../components/Show';
-import LinkTypes from '../sandboxes/visx-linktypes/Example';
-import LinkTypesSource from '!!raw-loader!../sandboxes/visx-linktypes/Example';
-import packageJson from '../sandboxes/visx-linktypes/package.json';
-
-const LinkTypesPage = () => {
-  return (
-    <Show
-      events
-      title="Link Types"
-      codeSandboxDirectoryName="visx-linktypes"
-      component={LinkTypes}
-      margin={{
-        top: 40,
-        left: 40,
-        right: 40,
-        bottom: 40,
-      }}
-      packageJson={packageJson}
-    >
-      {LinkTypesSource}
-    </Show>
-  );
-};
-export default LinkTypesPage;
+import React from 'react';
```

Comment:
i don't understand this diff
![image](https://user-images.githubusercontent.com/7409244/133530822-1ffc04e2-da1f-4836-8466-9446f6175441.png)